source/extensions/filters/http/basic_auth/basic_auth_filter.cc

```diff
@@ -111,7 +111,10 @@ Http::FilterHeadersStatus BasicAuthFilter::onDenied(absl::string_view body,
       Http::Code::Unauthorized, body,
       [this](Http::ResponseHeaderMap& headers) {
         const auto request_headers = this->decoder_callbacks_->requestHeaders();
-        const std::string uri = Http::Utility::buildOriginalUri(*request_headers, MaximumUriLength);
+        const std::string uri =
```

Comment:
Hmm... I wonder if there can be a case (in a non-unit-test) that has an empty request headers table.
I can only see this with tcp_proxy, but wonder if I'm missing anything.